### PR TITLE
Sync and update golangci-lint configurations

### PR DESCRIPTION
**Pull request by AlekSi**
## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [x] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by chilagrow — APPROVED** [review]

File: integration/.golangci.yml [hunk 1]
```diff
@@ -59,7 +65,7 @@ linters-settings:
         alias: dto
   # ineffassign
   lll:
-    line-length: 130
+    line-length: 175 # TODO reduce it to 130
```
Comment (noisersup) [inline integration/.golangci.yml:68]: is that our new goal? :slightly_smiling_face:

Comment (noisersup) [inline integration/.golangci.yml:68]: Is there any background behind that? Do we have any place in code which requires as to adjust it to 175?

**Review by noisersup — APPROVED** [review]
Overall looks good to me. One question aside

File: integration/.golangci.yml (hunk 1 above)
Comment (AlekSi) [reply integration/.golangci.yml:68]: Yep, we have long lines in our integration tests. Before that PR, they were not checked due to mistake in the configuration

**Comment by AlekSi** [comment]
There is a bug, will fix after release

**Review by chilagrow — APPROVED** [review]

**Review by noisersup — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
LGTM

**Pull request merged**